libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -86,6 +106,22 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
     }
   }
 
+  @HostListener('window:keyboardWillShow')
```

Comment:
Tests?

Reply:
I believe that the `keyboardWillShow`-event is dispatched by cordova (capacitor), but won't we experience "the same wierdness" when running on a mobile browser?

Reply:
In the case, that this is also a problem for mobile screen keyboards, it seems that the resize event will be an option:
https://stackoverflow.com/questions/27132796/is-there-any-javascript-event-fired-when-the-on-screen-keyboard-on-mobile-safari

Reply:
Added 👌